src/main/java/hudson/plugins/analysis/core/AbstractResultAction.java

```diff
@@ -104,12 +106,12 @@ public ToolTipProvider getToolTipProvider() {
      *
      * @return the associated build of this action
      */
-    public final AbstractBuild<?, ?> getOwner() {
+    public final Run<?, ?> getOwner() {
```

Comment:
I definitely know one internal plugin, which will fail due to the invalid list cast after such upgrade. For me it should be introduced in a 2.0 version